tests/controls/webtest.cpp

```diff
@@ -302,6 +306,12 @@ void WebTestCase::SetPage()
     CPPUNIT_ASSERT_EQUAL("other text", m_browser->GetPageText());
 }
 
+void WebTestCase::EmulationLevel()
+{
+    CPPUNIT_ASSERT(m_browser->SetEmulationLevel(wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT));
+    CPPUNIT_ASSERT_EQUAL(wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT, m_browser->GetEmulationLevel());
```

Comment:
It is just for test_gui.exe, not for every program. So, it is okay, but I could do it.